### DISCUSSION TITLE
Try to flash ESP32-S3-Box3

### DISCUSSION THREAD

**Discussion by geheim27** [toverainc/willow#280] [Q&A]
Hi,

I'm trying to flash a ESP32-S3-Box**3**.  The Web Flasher seems to detect a ESP32-S3-Box and it's not possible to change. If I flash ESP32-S3-Box image, i end up with an white screen.

Regards and thanks for help.

**Comment by kristiankielhofner** [comment]
To flash the BOX-3 you will need to use our preview web flasher:

https://flash-preview.heywillow.io/

The BOX-3 devices are fairly new and we would love to hear you feedback on how this works for you!

**Reply by psilantropy** [reply]
![image](https://github.com/toverainc/willow/assets/79955883/0ce56b9d-d0de-452d-8ae4-b4c0ddaa334f)

Hi there. Thanks for the link. Is the firmware missing? Not currently possible to flash.
Refresh cached and it sits on "fetching your willow release" but never completes.

I was getting "Error: CryptoJS is not defined" is the other flash page before trying this one out. I get that error if I attempt to use the other device types also.

**Reply by psilantropy** [reply]
I tried in Edge instead with the non-preview web flasher. Flash progressing now.
Edit all working now, great job!

**Comment by geheim27** [comment]
Thanks for the link. I've successful flashed the device. I'm using German language (https://wisng.tovera.io/api/willow?force_language=de) and Alexa keyword.
In general, it's quite impressive, especially the response time.
Keyword isn't always detected. Also STT is sometimes faulty e.g words not separated or separated wrong. Maybee due to German language? Will try to do more testing in the next days/weeks.

**Reply by kristiankielhofner** [reply]
There are some configuration options for wake word sensitivity, you can try 2CH_95 under Advanced Configuration with WAS. 2CH_95 will be the default in the next release. One of the issues we've noted with wake words is sensitivity can be influenced by accent as the wake words are trained on a variety of native (mostly American) English speakers. When we start the sample collection process for our "Hey Willow" wake word we will make sure to collect training samples for non-native and accented speakers. This sample selection will involve the community so once we announce this process it would be great if you could participate with some samples of your own voice!

While none of the Willow developers speak German we've had positive reports of people using German. German is a "top 5" supported language in terms of training data with the Whisper implementation we are using so it should be well supported (the word error rates are similar to English). Note that with language support we use an accelerated version of the same Whisper implementations most other projects use. One thing you can try is using a different model and accuracy settings. For example:

`https://wisng.tovera.io/api/willow?force_language=de&model=large&beam_size=5`

Somewhat paradoxically we've also found that in some cases larger models and higher accuracy settings can actually be "too smart for their own good" with speech commands and English (what we test most with). I'm not sure if this is the case with German and your specific commands but you can also try smaller models as these can actually provide more accurate transcriptions depending on a variety of factors. Example:

`https://wisng.tovera.io/api/willow?force_language=de&model=small&beam_size=2`
